README.md

```diff
@@ -1,2 +1,145 @@
 # redux-needs &middot; [![Build Status](https://travis-ci.org/LUKKIEN/redux-needs.svg?branch=master)](https://travis-ci.org/LUKKIEN/redux-needs) [![codecov](https://codecov.io/gh/LUKKIEN/redux-needs/branch/master/graph/badge.svg)](https://codecov.io/gh/LUKKIEN/redux-needs)
 
+> 🎁 Wrapping everyday data to your containers in need 🎁
+
+Bind actions to changes in your redux state based on the needs of your active
+redux components.
+
+[Usage](#usage) |
+[API](#api) |
+[Installation](#installation) |
```

Comment:
I put it here because that's the way the [common-readme](https://github.com/noffle/common-readme) template does it and also I think most people will find the library through npm or yarn anyway, making installation a bit of a no-brainer. Having said that, I feel neutral about it, if you prefer it at the top, I'll move it to the top.